### ISSUE TITLE
inconsistent output of first column (source IP: src_ip)

### ISSUE THREAD

**Issue by jesteria** [nprint/nprint#21]
nPrint v1.1.7 appears to arbitrarily omit the contents of the first column of output, namely the source IP or `src_ip`.

(As a result, runs of nprintML using the "index" aggregator arbitrarily and inconsistently fail, because nPrint data cannot be joined with labels indexed by source IP.)

---

For example, the first execution works:

```
$ nprint -t4 -P test/data/dir-pcap/pcaps/encrypted/encrypted_90.pcap | head -c20
src_ip,ipv4_ver_0,ip
```

But the _very next_ execution fails:

```
$ nprint -t4 -P test/data/dir-pcap/pcaps/encrypted/encrypted_90.pcap | head -c20
,ipv4_ver_0,ipv4_ver
```

This phenomenon does appear, however, to be **arbitrary**. The next 10 runs or so succeeded, before another failure:

```
$ nprint -t4 -P test/data/dir-pcap/pcaps/encrypted/encrypted_90.pcap | head -c20
src_ip,ipv4_ver_0,ip
$ nprint -t4 -P test/data/dir-pcap/pcaps/encrypted/encrypted_90.pcap | head -c20
src_ip,ipv4_ver_0,ip

…

$ nprint -t4 -P test/data/dir-pcap/pcaps/encrypted/encrypted_90.pcap | head -c20
,ipv4_ver_0,ipv4_ver
```

---

For further clarity, below is the same, but with all (two) lines of output, (only their columns snipped for legibility).

A success:

```
$ nprint -t4 -P test/data/dir-pcap/pcaps/encrypted/encrypted_90.pcap | cut -b1-20
src_ip,ipv4_ver_0,ip
74.189.204.184,0,1,0
```

Immediately followed by a failure:

```
$ nprint -t4 -P test/data/dir-pcap/pcaps/encrypted/encrypted_90.pcap | cut -b1-20
,ipv4_ver_0,ipv4_ver
,0,1,0,0,0,1,0,1,0,0
```

**Comment by jesteria** [comment]
I can confirm that this behavior appears to be redressed by branch `config-fix` (head 739fc7e17aa9ad74a2dcb4423073bd0982ca6386) :+1:.

The following adhoc test fails consistently against v1.1.7 (installed into the PATH):

```
$ result=passed;
  for run in {1..50}; do
  if test "$(nprint -t4 -P ~/dev/repo/cdac/nprintml/test/data/dir-pcap/pcaps/encrypted/encrypted_90.pcap | head -c1)" == ","; then
    result=failed;
    echo fail;
  else
    echo pass;
  fi;
  done;
  echo $result
pass
pass
pass
pass
pass
pass
pass
pass
pass
fail
fail
…
pass
failed
```

The same test consistently passes against nPrint built from this branch's source:

```
$ result=passed;
  for run in {1..50}; do
  if test "$(/tmp/nprintfix/bin/nprint -t4 -P ~/dev/repo/cdac/nprintml/test/data/dir-pcap/pcaps/encrypted/encrypted_90.pcap | head -c1)" == ","; then
    result=failed;
    echo fail;
  else
    echo pass;
  fi;
  done;
  echo $result
pass
pass
pass
pass
pass
pass
pass
pass
pass
pass
pass
…
pass
pass
passed
```

**Comment by JordanHolland** [comment]
Closed with #23
